### DISCUSSION TITLE
Proposals: Group sorting, Gameplay leaderboard, Collection list numbers

### DISCUSSION THREAD

**Discussion by gamedestroyer222** [ppy/osu#26008] [Ideas] [answered]
Can we bring back group sorting since it was very helpful when combined with normal sorting on stable.

Can we have a toggle for the in-game leaderboard so we can see the other players score and not just their placement number and profile picture. Since yeah we can hold ctrl to see the rest of it but can we have a toggle that way we don't have to hold it in order to see the rest of it (Also side note would be nice for leaderboard to be movable in the skin editor)

Can we have it to where the collection list shows how many maps there are in each collection like how it is for stable so that way we don't have to click each individual one in order to see how many is in them
![image](https://github.com/ppy/osu/assets/45153456/55a4f697-c132-4218-87de-aa0d921132c4) Laser
![image](https://github.com/ppy/osu/assets/45153456/b0a1e7fe-ac01-45b6-a9f1-3974ed3dff6b) Stable

**Comment by bdach** [comment, marked as answer]
all three items have been proposed at one point or another in pre-existing discussions

**Comment by gamedestroyer222** [comment]
I guess I must have missed them since I did try searching if these were already proposed but yeah thank you for answering though
